kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -268,6 +284,29 @@
         const newMsg = createReturnMsg({ message, data: __version });
         return this.hashi.mediator.sendMessage(newMsg);
       },
+      sendChannelMetadata(message) {
+        return ChannelResource.fetchModel({ id: this.topic.channel_id })
+          .then(channel => {
+            return createReturnMsg({
+              message,
+              data: {
+                  id: channel.id,
+                  title: channel.title,
+                  description: channel.description,
+                  thumbnail: channel.thumbnail,
+                  availableAuthors: channel.available_authors,
+                  availableTags: channel.available_tags,
+                  availableActivityTypes: channel.available_activity_types,
```

Comment:
For better API design I think this should be a separate method. I imagine 2 methods:
- getChannelMetadata -- title, description, thumbnail
- getChannelFilterOptions -- availableAuthors, availableTags, availableActivityTypes

Then if a custom presentation doesn't need filters, the second query is not done. What do you think?